src/pages/index.en.tsx

```diff
@@ -24,6 +24,95 @@ const shuffleArray = (array: any[]) =>
     .sort((a, b) => a.sort - b.sort)
     .map(({ value }) => value);
 
+type ProductCardInfo = {
+  productName: string;
+  title: string;
+  descriptionText: {
+    json: any;
+  };
+  location: string;
+  language: string[];
+  button: {
+    title: string;
+    link: string;
+  };
+};
+
+const ProductCard: React.FC<ProductCardInfo> = (props) => (
+  <div className="column is-4 is-12-mobile">
+    <div className="jf-card has-background-white p-8 p-6-mobile">
+      <div className="eyebrow is-small mb-5 mb-4-mobile">
+        {props.productName}
+      </div>
+      <h3 className="mb-6 mb-5-mobile">{props.title}</h3>
+      <div className="mb-6 mb-5-mobile">
+        {documentToReactComponents(props.descriptionText.json)}
+      </div>
+      <div className="mt-auto">
+        <div className="mb-6">
+          {props.location} · {props.language.join(" · ")}
+        </div>
```

Comment:
In the mocks this looks like a unique text style outside the was we implemented for the design system (small caps) and it is in dark-grey instead of black.